sound/hybrid_font.h

```diff
@@ -138,25 +138,48 @@ class HybridFont : public SaberBase {
       PlayPolyphonic(effect);
     }
   }
-  
-  void StartSwing() override {
-    if (!guess_monophonic_) {
-      if (swing_player_) {
-        // avoid overlapping swings, based on value set in ProffieOSSwingOverlap.  Value is
-        // between 0 (full overlap) and 1.0 (no overlap)
-        if (swing_player_->pos() / swing_player_->length() >= config_.ProffieOSSwingOverlap) {
-          swing_player_->set_fade_time(swing_player_->length() - swing_player_->pos());
-          swing_player_->FadeAndStop();
-          swing_player_.Free();
-          swing_player_ = PlayPolyphonic(&swng);
+
+  void StartSwing(const Vec3& gyro, float swingThreshold, float slashThreshold) override {
+    Vec3 gyro_slope = fusor.gyro_slope();
+    //Radians per second per second
+    float rss = sqrtf(gyro_slope.z * gyro_slope.z + gyro_slope.y * gyro_slope.y) * (M_PI/180);
+    float swing_speed = fusor.swing_speed();
+    if (swing_speed > swingThreshold) {
+      if (!guess_monophonic_) {
+        if (swing_player_) {
+          // avoid overlapping swings, based on value set in ProffieOSSwingOverlap.  Value is
+          // between 0 (full overlap) and 1.0 (no overlap)
+          if (swing_player_->pos() / swing_player_->length() >= config_.ProffieOSSwingOverlap) {
+            swing_player_->set_fade_time(swing_player_->length() - swing_player_->pos());
+            swing_player_->FadeAndStop();
+            swing_player_.Free();
+          }
         }
+        if (!swing_player_) {
+          if (!swinging_) {
+            if (rss > slashThreshold && slsh.files_found()) {
+              swing_player_ = PlayPolyphonic(&slsh);
```

Comment:
before we check this in; there is an alternate way that may potentially be interesting....
Basically, we could pick which swing sound to play based on the slope.
So instead of having separate swings and slashes, we would need a progression of swing sounds, and we would pick a sound based on how "aggressive" the swings are.
WDYT?

Reply:
That is an interesting notion.  It brings to mind another idea that I've had in my head for a while but never really tried to implement: a third looped hum.  Rather than having a pair that transitions from A - B, this third hum will increase in volume (and pitch) as the saber swings faster to provide an overlay that, if it works like I think it might, could potentially do away with static sounds altogether.

In any event, there are plenty of fonts that exist right now that can take advantage of slsh so I'd like to get this checked in as is and, after I do a couple other things, come back to thinking about how we either: a) implement a progressive set of static sounds or b) look at another looped sound that reacts differently enough to the movement of the saber that it can function as an accent that changes enough that we don't need static sounds.

Reply:
sounds good to me.